tests/test_soak.py

```diff
@@ -0,0 +1,109 @@
+import dcos.metronome
+import dcos.package
+import json
+import os
+import logging
+import pytest
+import shakedown
+
+import utils
+from utils import SPARK_PACKAGE_NAME
+
+
+LOGGER = logging.getLogger(__name__)
+TERASORT_JAR='https://downloads.mesosphere.io/spark/examples/spark-terasort-1.0-jar-with-dependencies_2.11.jar'
+TERASORT_MAX_CORES=6
+
+
+def setup_module(module):
+    _require_spark_cli()
+
+
+@pytest.mark.soak
+def test_terasort():
+    if utils.hdfs_enabled():
+        _delete_hdfs_terasort_files()
+        _run_teragen()
+        _run_terasort()
+        _run_teravalidate()
+
+
+def _require_spark_cli():
```

Comment:
Do you still want to combine this method with require_spark(), which is in another file?